modules/dailymotionBidAdapter.js

```diff
@@ -72,4 +75,28 @@ export const spec = {
   interpretResponse: serverResponse => serverResponse?.body ? [serverResponse.body] : [],
 };
 
+function getVideoMetadata(bidRequest) {
+  const videoAdUnit = deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = deepAccess(bidRequest, 'params.video', {});
+
+  const videoParams = {
+    ...videoAdUnit,
+    ...videoBidderParams // Bidder Specific overrides
+  };
+
+  const videoMetadata = {
+    description: videoParams.hasOwnProperty('description') ? videoParams.description : '',
+    duration: videoParams.hasOwnProperty('duration') ? videoParams.duration : 0,
+    iabcat2: videoParams.hasOwnProperty('iabcat2') ? videoParams.iabcat2 : '',
+    id: videoParams.hasOwnProperty('id') ? videoParams.id : '',
+    lang: videoParams.hasOwnProperty('lang') ? videoParams.lang : '',
+    private: videoParams.hasOwnProperty('private') ? videoParams.private : false,
+    tags: videoParams.hasOwnProperty('tags') ? videoParams.tags : '',
+    title: videoParams.hasOwnProperty('title') ? videoParams.title : '',
+    topics: videoParams.hasOwnProperty('topics') ? videoParams.topics : '',
```

Comment:
I feel like using `hasOwnProperty` is a bit verbose. You could just use `||` for each case when property does not exist or value is null/undefined
<img width="234" alt="Capture d’écran 2024-02-23 à 11 10 21" src="https://github.com/dailymotion-oss/Prebid.js/assets/11768029/fd685d89-87e4-49bd-b578-edd7d7d06394">

Reply:
done